latch_cli/snakemake/config/parser.py

```diff
@@ -81,10 +81,16 @@ def generate_metadata(
 
         is_file = typ in {LatchFile, LatchDir}
         param_typ = "SnakemakeFileParameter" if is_file else "SnakemakeParameter"
+
+        def _best_effort_display_name(param_name: str):
+            if param_name[0] == "_":
+                param_name = param_name[1:]
+            return " ".join(map(lambda x: x.capitalize(), param_name.split("_")))
```

Comment:
```suggestion
        expr = re.compile(r"_+")

        def title_case(x: str):
            return expr.sub(" ", x).title().strip()
```
we should also prob have support for camelcase config options but nbd rn

Reply:
also put this function in serialize utils